### PR TITLE
Added title and description fields with option to show on tile

### PR DESCRIPTION
**Pull request by agitator**

### CODE REVIEW

**Comment by tomgross** [comment]
In the schema there are properties _show_title_ and _show_description_ which are never used in the code or template. In the template the condition is on the title vs. description property. This makes sense in my opinion. I propose to remove the _show_title_ and _show_description_ properties from the schema.

**Comment by jensens** [comment]
@tomgross indeed this is confusing

I am just not sure if the boolean fields are meant for the listing itself, @agitator how is it meant?

**Comment by agitator** [comment]
dooh ... fixed now

File: plone/app/standardtiles/contentlisting.py [hunk 1]
```diff
@@ -149,6 +179,20 @@ def update(self):
             self.sort_order = 'ascending'
         self.view_template = self.data.get('view_template')
 
+    @property
+    def title(self):
+        if self.data.get('show_title'):
```
Comment (jensens) [inline plone/app/standardtiles/contentlisting.py]: and if title is empty?

File: plone/app/standardtiles/contentlisting.py [hunk 2]
```diff
@@ -149,6 +179,20 @@ def update(self):
...
+            return self.data.get('title')
+        else:
+            return None
+
+    @property
+    def description(self):
+        if self.data.get('show_description'):
```
Comment (jensens) [inline plone/app/standardtiles/contentlisting.py]: same here

File: plone/app/standardtiles/contentlisting.py (hunk 1 above)
Comment (agitator) [inline plone/app/standardtiles/contentlisting.py]: then it just returns None?

Comment (agitator) [inline plone/app/standardtiles/contentlisting.py]: or should it explicitly be 

`if self.data.get('show_title') and self.data.get('title'):`

**Comment by agitator** [comment]
@tomgross idea was to explicitly turn on/off the title/description but the longer I think about it, leaving it empty could be enough

@datakurre what do you think?

**Comment by agitator** [comment]
removed show boolean, will show now if filled in

**Pull request merged**